apps/sms/test/marionette/new_activity_test.js

```diff
@@ -102,7 +102,7 @@ marionette('Messages as "new" activity target', function() {
         return !!window.wrappedJSObject.Compose;
       }), 'XSS should not be performed');
 
-      assert.isTrue(messagesApp.Composer.sendButton.enabled());
+      client.waitFor(() => messagesApp.Composer.sendButton.enabled());
```

Comment:
Looks like `launchAsActivity` already has `switchTo` inside and it should wait for body element with app ready class. If it still not enough, maybe we could add a class to composer to represent composer status is ready?